### ISSUE TITLE
Feature request: Delete recent option

### ISSUE THREAD

**Issue by sphh** [m-i-n-a-r/randomix#64]
It would be great, if it were possible to delete entries from the list of recent options.

Here is a use case: I use your fabulous app to select one out of many items. Once in a while, I want to add another item or delete one. I then have two entries in the list of recent options: The original one and the edited one. You then have to check very closely to select the last updated one. Here it would be great, if the original option could be deleted.

**Comment by m-i-n-a-r** [comment]
You can swipe them left or right to delete them, or long press on them to pin them! Try and let me know if it works for you :)

**Comment by sphh** [comment]
Yes, that works. I now also found (after I searched for it), that this is mentioned in the README, but without a note, how this is done. Could this be added to the README?

**Comment by m-i-n-a-r** [comment]
Yep, I'll update the readme as soon as I start working on the next update (end of July probably). Thanks for suggesting it!
